src/actions/importing/deactivate-conflicting-plugins-action.php

```diff
@@ -43,9 +48,21 @@ class Deactivate_Conflicting_Plugins_Action extends Abstract_Importing_Action {
 	/**
 	 * Class constructor.
 	 *
-	 * @param Conflicting_Plugins_Service $conflicting_plugins_service The Conflicting plugins Service.
+	 * @param Indexable_Repository         $indexable_repository        The indexables repository.
+	 * @param wpdb                         $wpdb                        The WordPress database instance.
+	 * @param Indexable_To_Postmeta_Helper $indexable_to_postmeta       The indexable_to_postmeta helper.
+	 * @param Options_Helper               $options                     The options helper.
+	 * @param Wpdb_Helper                  $wpdb_helper                 The wpdb_helper helper.
+	 * @param Conflicting_Plugins_Service  $conflicting_plugins_service The Conflicting plugins Service.
 	 */
-	public function __construct( Conflicting_Plugins_Service $conflicting_plugins_service ) {
+	public function __construct(
+		Indexable_Repository $indexable_repository,
+		wpdb $wpdb,
+		Indexable_To_Postmeta_Helper $indexable_to_postmeta,
+		Options_Helper $options,
+		Wpdb_Helper $wpdb_helper,
+		Conflicting_Plugins_Service $conflicting_plugins_service ) {
```

Comment:
It might be out of scope for this particular issue, but the Abtract_Importing_Action doesn't use most of these dependencies. Only the Options_Helper is used. Perhaps it's worth considering removing those dependencies there and add them where they're actually used.

Reply:
Yeap, makes sense @diedexx thanks! Will do that and ping you for approval after